contracts/contracts/strategies/AaveStrategy.sol

```diff
@@ -11,9 +11,47 @@ import {
     InitializableAbstractStrategy
```

Comment:
Is there any merit in leaving this strategy untouched as a V1 integration and creating a new file AaveStrategyV2 or similar?

Reply:
Only if we thought it likely that we would use AAVE v1 again. But we can aways resurrect this from the Git history if we really need it.